### ISSUE TITLE
Connection Migration section should talk about client and server, not endpoint and peer

### ISSUE THREAD

**Issue by marten-seemann** [quicwg/base-drafts#4125]
We decided to only specify client-initiated connection migration in QUIC v1. The section about Connection Migration mostly talks about endpoint and peer though, which makes it really hard to parse. I haven't implement migration yet in quic-go, and reading this section confused me at multiple places.

I believe that specifying the role of the endpoint would make things a lot clearer, and could offer to come up with a PR.

**Comment by martinthomson** [comment]
I don't think this is worth doing, but would look at a PR.

**Comment by ianswett** [comment]
To clarify, this would only apply to connection migration, not the path validation section that immediately precedes it.

**Comment by janaiyengar** [comment]
@marten-seemann: The idea was to keep the migration mechanisms generic, since they _can_ be applied to either endpoint. However, this isn't adequate if we support migration at both ends, since you'd have to deal with other situations, such as simultaneous migration. v1 simply limits the scope of where these mechanisms ought to be applied.

There's value in that, but I understand your reservations. Like @martinthomson, I'm not keen to follow the premise of this issue, but I'd be open to a PR. I'd also be open to clarifying the parts you found confusing.

**Comment by ianswett** [comment]
Looking at that section again, I don't believe I want to do anything about this issue.

**Comment by martinthomson** [comment]
This choice was deliberate.  Maybe it's nicer, but now is not really a good time.
